camera_aravis2/include/camera_aravis2/camera_xml_exporter.h

```diff
@@ -60,20 +60,20 @@ class CameraXmlExporter : public CameraAravisNodeBase
      */
     virtual ~CameraXmlExporter();
 
-  protected:
-    /**
-     * @overload
-     * @brief Set the up launch parameters.
-     */
-    void setupParameters() override;
-
     /**
      * @brief Export XML data from camera and write to file.
      *
      * @return True if successful. False, otherwise.
      */
     [[nodiscard]] bool export_xml_data_to_file();
```

Comment:
completely unrelated, but is it intentional that you mix snake_case (this method) with camelCase (all other methods)?

Reply:
No, that is not intended. Thank you for point that out.